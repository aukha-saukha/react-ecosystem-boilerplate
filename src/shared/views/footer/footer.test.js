import { shallow } from 'enzyme';
import React from 'react';

import Footer from './footer.react';

describe('Footer component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
