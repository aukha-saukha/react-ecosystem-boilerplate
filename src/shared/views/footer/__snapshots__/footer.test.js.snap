// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Renders correctly 1`] = `
<footer
  className="display-flex flex-items-align-center padding-all-15 footer"
>
  <nav>
    <ul
      className="display-flex"
    >
      <li
        key="About us"
      >
        <NavLink
          className="links"
          to="/about-us"
        >
          About us
        </NavLink>
      </li>
      <li
        key="Contact us"
      >
        <NavLink
          className="links"
          to="/contact-us"
        >
          Contact us
        </NavLink>
      </li>
    </ul>
  </nav>
</footer>
`;
